### ISSUE TITLE
[09:42:50] 	QPDAL_IO_PLUGIN.dll does not seem to be a valid plugin	(Cannot load library C:\Program Files\CloudCompareProjects\CloudCompare\plugins\QPDAL_IO_PLUGIN.dll: 找不到指定的程序。)

### ISSUE THREAD

**Issue by sphugh** [CloudCompare/CloudCompare#1382]
An error occurred while compiling CloudCompare (v2.11 +).I've tried everything available online, such as configuring system environment variables, adding as many dependencies as possible, and nothing has solved this problem.Please provide us with some possible information.

**Comment by WargodHernandez** [comment]
are the PDAL dependencies being placed into `C:\Program Files\CloudCompareProjects\CloudCompare\` and not `C:\Program Files\CloudCompareProjects\CloudCompare\plugins\`?

**Comment by kai156277** [comment]
I have same the same problem. 

My Version is : 2.11.3 [Windows 64-bit] Compiled with MSVC 1916 and Qt 5.9.9。

pdal use conda install. only pdal_util.dll and pdalcpp.dll is under `E:\Program Files\CloudCompare-2.11.3\CloudCompare\`。

Have I lost any other dependencies?

![root_path](https://user-images.githubusercontent.com/16378876/104433391-040fe180-55c5-11eb-82b4-be3cfe897415.png)
![plugins_path](https://user-images.githubusercontent.com/16378876/104433406-070ad200-55c5-11eb-9720-5f90caa2a976.png)

**Comment by tmontaigu** [comment]
Yes, PDAL relies on more dependencies than what seems to be copied in your CloudCompare folder.

To work around this you can try to launch CloudCompare from a terminal with the conda environment you installed pdal in
activated/

**Comment by kai156277** [comment]
Thanks! I add `path` to env, then las can load。
